tests/integration/ha_tests/test_self_healing.py

```diff
@@ -56,8 +55,8 @@
 logger = logging.getLogger(__name__)
 
 APP_NAME = METADATA["name"]
-PATRONI_PROCESS = "/snap/charmed-postgresql/[0-9]*/usr/bin/patroni"
-POSTGRESQL_PROCESS = "/snap/charmed-postgresql/current/usr/lib/postgresql/14/bin/postgres"
+PATRONI_PROCESS = "patroni"
+POSTGRESQL_PROCESS = "postgres"
```

Comment:
IIRC there are a number of processes named postgres running, but if we turn out to be hitting the wrong process we can tweak that later down the road.

Reply:
Yeah, you're right Dragomir. There are more processes that match when we search for `postgres`. I double checked with Mykola and in this particular case we should kill all of them.